z2/src/docgen.rs

```diff
@@ -144,6 +163,56 @@ fn insert_key(val: &mut serde_yaml::Value, components: &Vec<String>, idx: usize,
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Hash, Eq)]
+pub enum ApiMethod {
+    JsonRpc { name: String },
+    Rest { uri: String },
+}
+
+pub fn get_implemented_jsonrpc_methods() -> Result<Vec<ApiMethod>> {
+    let mut methods = Vec::new();
+    // Construct an empty node so we can check for the existence of RPC methods without constructing a full node.
+    let n0 = SecretKey::new()?;
+    let genesis_accounts: Vec<(Address, String)> = vec![
+        (
+            address!("7E5F4552091A69125d5DfCb7b8C2659029395Bdf"),
+            "5000000000000000000000".to_string(),
+        ),
+        // privkey db11cfa086b92497c8ed5a4cc6edb3a5bfe3a640c43ffb9fc6aa0873c56f2ee3
+        (
+            address!("cb57ec3f064a16cadb36c7c712f4c9fa62b77415"),
+            "5000000000000000000000".to_string(),
+        ),
+    ];
```

Comment:
FWIW I'd prefer
```suggestion
    let genesis_accounts: Vec<(Address, String)> = vec![];
```
and removing https://github.com/Zilliqa/zq2/blob/96cd025e0f1efb5fe11e9aa9292183e101d17ab4/zilliqa/src/state.rs#L87-L89

There's no reason networks without any genesis accounts shouldn't be valid (except that on they aren't very useful).

Reply:
Cool! That's been annoying me for a while, so will address in a future PR - https://github.com/Zilliqa/zq2/issues/932